nautilus_trader/adapters/binance/futures/providers.py

```diff
@@ -100,18 +139,30 @@ async def load_ids_async(
         self._log.info(f"Loading instruments {instrument_ids}{filters_str}.")
 
         # Extract all symbol strings
-        symbols: list[str] = [
-            instrument_id.symbol.value.replace("-PERP", "") for instrument_id in instrument_ids
-        ]
+        symbols = [instrument_id.symbol.value for instrument_id in instrument_ids]
```

Comment:
Are we changing the symbols for perpetuals? The reason I went with this was to distinguish between Spot/Margin and Futures instruments.

Reply:
Yep, the functionality is still there, this is handled by the [`BinanceSymbol`](https://github.com/nautechsystems/nautilus_trader/pull/973/files#diff-828b583ba79fa3c4289ba81ec865cb152747d5edb5df7c2685ee4bac46c53c70) class/type. There was an effort to push all conversion between internal and Binance symbols to the `HttpApi` & `WebsocketClient` layers but there are still a few exceptions.